src/Utils.js

```diff
@@ -167,6 +143,17 @@ function arraysEqual(match, left, right) {
   return left.length === right.length && left.every((el, i) => match(el, right[i]));
 }
 
+function childrenToArray(children) {
+  // NOTE(lmr): we currently use this instead of Children.toArray(...) because
+  // toArray(...) didn't exist in React 0.13
+  const result = [];
+  React.Children.forEach(children, (el) => {
```

Comment:
yes, but I was having trouble doing this properly. There are still several parts of enzyme that will depend on react for now. Removing that dependency is still probably something we want to do, but I'm considering out of the scope of this PR.